lib/text_chunker/strategies/recursive_chunk/separators.ex

```diff
@@ -157,14 +137,38 @@ defmodule TextChunker.Strategies.RecursiveChunk.Separators do
       "  while ",
       "  switch ",
       "  case ",
-      "  default ",
-      "\n\n",
-      "\n",
-      " "
-    ]
+      "  default "
+    ] ++ empty_and_new_line_separators()
+  end
+
+  def get_separators(:html) do
+    [
+      "<h1",
+      "<h2",
+      "<h3",
+      "<h4",
+      "<h5",
+      "<h6",
+      "<p",
+      "<ul",
+      "<ol",
+      "<li",
+      "<article",
+      "<section",
+      "<table"
+    ] ++
```

Comment:
I'm curious, why is it splitting on `<h1` rather than `<h1>`?

Reply:
I was just being lazy and following the other examples (vue).

Reply:
It just struck me while driving that this will also split on any HMTL tags that have attrs! So it makes way more sense than adding in the closing `>`.